tethys_apps/admin.py

```diff
@@ -149,18 +149,6 @@ def has_add_permission(self, request, obj=None):
         return False
 
 
-class CustomUser(UserAdmin):
-    def add_view(self, *args, **kwargs):
-        if UserQuotasSettingInline in self.inlines:
-            self.inlines.pop(self.inlines.index(UserQuotasSettingInline))
-        return super().add_view(*args, **kwargs)
-
-    def change_view(self, *args, **kwargs):
-        if UserQuotasSettingInline not in self.inlines:
-            self.inlines.append(UserQuotasSettingInline)
-        return super().change_view(*args, **kwargs)
-
```

Comment:
This defines a custom admin view for the User model and adds the field for controlling quotas on a per-user basis:

![image](https://user-images.githubusercontent.com/5123221/181092435-d10f48e8-aa5b-48da-bfc6-82ae75e7c00c.png)

Reply:
Removing this code would mean removing this feature. I'll take a look at it to see if I can figure out what is going on.